src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageReferenceCommands/AddPackageReferenceCommandRunner.cs

```diff
@@ -101,6 +101,7 @@ public async Task<int> ExecuteCommand(PackageReferenceArgs packageReferenceArgs,
             }
 
             var originalPackageSpec = matchingPackageSpecs.FirstOrDefault();
+            packageReferenceArgs.Logger.LogWarning("CentralPackageVersionsEnabled: " + originalPackageSpec.RestoreMetadata.CentralPackageVersionsEnabled.ToString());
```

Comment:
This warning message is not required because we log additional messages after updating props file and project file.

Reply:
To me, this looks like debug tracing, and should be removed before the PR is merged. From a customer experience point of view, it's not a warning, especially if it's always shown, true or false.

Reply:
Agreed, this has been removed.

Reply:
> IMO, this warning message is not required because we log additional messages after updating props file and project file.

I originally didn't have any additional log messages, but I have now added them.